src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -23,7 +23,7 @@ import NamexServices from '@/services/namex.services'
 import { Location, NrType, RequestCode } from '@/enums'
 
 @Component({})
-export default class ReserveSubmitButton extends Vue {
+export default class ReserveSubmit extends Vue {
```

Comment:
Fixed typo. (This component was always imported as "ReserveSubmit" so it didn't matter, but now it's correct and you can import the default name correctly.)